### PR TITLE
stop instance service using fully configured Retrofit client

### PR DESCRIPTION
**Pull request by robfletcher**
The tasks that use these services do their own timeout management.

### CODE REVIEW

**Comment by robfletcher** [comment]
PTAL @ajordens

**Comment by ajordens** [comment]
Took a look ... I believe the default `OkClient` has retryOnConnectionFailure = true.

In a pinch you could inject `OkHttpClientConfiguration` and just new OkClient(okHttpClientConfiguration.create()).

**Comment by robfletcher** [comment]
ah, right. I should have checked that. I'll update the PR

File: build.gradle [hunk 1]
```diff
@@ -38,7 +38,11 @@ allprojects {
       exceptionFormat = 'full'
     }
   }
-  
+
+  repositories {
+    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
```
Comment (ajordens) [inline build.gradle]: Don't need this eh?

**Comment by ajordens** [comment]
LGTM

**Pull request merged**